website/docs/documentation/Pipelines/CEMBA_MethylC_Seq_Pipeline/README.md

```diff
@@ -1,8 +1,10 @@
+# CEMBA Pipeline Overview
+
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
 | [CEMBA_v1.0.0](https://github.com/broadinstitute/warp/releases) | July 28, 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
 
-# Table of Contents
+## Table of Contents
```

Comment:
Great question! I preserved all TOCs while fixing the formats, but in the future, just to make our own life easier, we could simply rely on Vuepress's built-in sidebar navigation, and even if we want to insert such a TOC, we could do `[[TOC]]` instead of manually crafting one, what do you think?